wallet/src/utils/tokens/index.js

```diff
@@ -84,11 +85,14 @@ export async function createAndInitializeMint({
   amount, // Number of tokens to issue
   decimals,
   initialAccount, // Account to hold newly issued tokens, if amount > 0
+  transactionCallback,
+  onSuccess = () => {},
+  onError = () => {},
 }) {
   let transaction = new Transaction();
   transaction.add(
     SystemProgram.createAccount({
-      fromPubkey: owner.publicKey,
+      fromPubkey: owner.address,
```

Comment:
I'm not sure If I'd introduce the cryptidAccount Layer in this file? Any reasons why you removed the mapping on layer up?

Reply:
Just that the lower down needs cryptid further down

Reply:
I fine with that I tried to be minimal invasive in that "legacy" code :) But that would not be a blocker to me.